backend/src/controllers/reports/Tab_18_rpt_P_StatusSummary.js

```diff
@@ -23,10 +23,17 @@ controller.Tab_18_rpt_P_StatusSummary = async (request, reply) => {
     // Get the data from the database.
     const getDate = async () => new Date();
 
+    const projectId = Number(request.query.project);
+    // Get the data from the database.
     const result = {
+      project: await model.getProjectById(projectId),
+      deliverables: await model.projectStatusReport(projectId),
+      milestones: await model.getMilestones(projectId),
+      alignment: await model.getStrategicAlignment(projectId),
+      statuses: await model.getProjectStatuses(projectId),
```

Comment:
nice. it would be cool if most controllers were this brief and clear.

there's probably a few opportunities to re-use existing models for upcoming controllers